lib/matplotlib/axes/_base.py

```diff
@@ -1002,7 +1002,7 @@ def _update_transScale(self):
 
     def get_position(self, original=False):
         """
-        Get a copy of the axes rectangle as a `.Bbox`.
+        Get a copy of the Axes rectangle as a `.Bbox`.
```

Comment:
```suggestion
        Returns a copy of the Axes rectangle as a `.Bbox`.
```

Reply:
Reworded. Copy is not the right word anyway.